spring-integration-hazelcast/README.md

```diff
@@ -200,6 +200,40 @@ Sample definition is as follows :
 **Reference :** http://docs.hazelcast.org/docs/3.4/manual/html-single/hazelcast-documentation.html#continuous-query 
 
 
+## HAZELCAST CLUSTER MONITOR INBOUND CHANNEL ADAPTER 
+
+Hazelcast Cluster Monitor enables to listen to the modifications performed on cluster. Hazelcast Cluster Monitor Inbound Channel Adapter is an event-driven inbound channel adapter and listens related Membership, Distributed Object, Migration, Lifecycle and Client events. Its basic definition is as follows : 
+```
+<int-hazelcast:cm-inbound-channel-adapter 
+				 channel="monitorChannel" 
+				 hazelcast-instance="instance" 
+				 monitor-types="MEMBERSHIP, DISTRIBUTED_OBJECT, MIGRATION, LIFECYLE, CLIENT" /> 
+```
+Basically, it requires four attributes as follows : 
+
+* **channel :** Specifies channel which message is sent. It is mandatory attribute. 
```

Comment:
No, it isn't mandatory, if you specify `id`. In that case the channel can be auto-created.
Of course this logic must be supported by parser. 
Will see.
From other side: it's OK. Let's leave your text here as is!